README.md

```diff
@@ -12,6 +12,8 @@ npm run compile:gameStateSchema
 npm run --silent dev
 ```
 
+Use `npm run clean:game` to delete old save files.
+
```

Comment:
How about:
```diff
-```
-
-Use `npm run clean:game` to delete old save files.
+# Delete the current save file
+npm run clean:game
+```
```

Reply:
updated